source/math/FloatSize3D.ooc

```diff
@@ -21,59 +21,59 @@ import IntSize3D
 use ooc-base
 
 FloatSize3D: cover {
-	width, height, depth: Float
-	volume ::= this width * this height * this depth
+	x, y, z: Float
+	volume ::= this x * this y * this z
 	length ::= this norm
-	empty ::= this width == 0 || this height == 0 || this depth == 0
-	norm ::= (this width squared() + this height squared() + this depth squared()) sqrt()
-	azimuth ::= this height atan2(this width)
+	empty ::= this x == 0 || this y == 0 || this z == 0
```

Comment:
That's a good question, these things appear in many places so we need some clarity here.